dev-tools/download_macos_deps.sh

```diff
@@ -25,7 +25,7 @@ DEST=/usr
 case `uname -m` in
 
     arm64)
-        ARCHIVE=local-arm64-apple-macosx11.1-3.tar.bz2
+        ARCHIVE=local-arm64-apple-macosx11.1-4.tar.bz2
```

Comment:
This file is in place now.

Reply:
This file is now updated to a version containing libraries for the correct architecture. 🤦